### PR TITLE
API endpoint corrections

### PR DESCRIPTION
**Pull request by MiketheBite**

### CODE REVIEW

File: pages/addNurse/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ export default function AddNursePage() {
 
   async function handleSubmitNurse(nurseData) {
     try {
-      const response = await fetch(`/api/nurses`, {
+      const response = await fetch(`/api/nurses/$[id]`, {
```
Comment (moonwave99) [inline pages/addNurse/index.js:9]: `$[id]` shouldn't be part of URL, see below

File: pages/api/nurses/[id].js [hunk 2]
```diff
@@ -31,9 +31,20 @@ export default async function handler(request, response) {
...
     } catch (error) {
-      response.status(500).json({ status: "Error updating nurse." });
+      return response.status(500).json({ status: "Error updating nurse." });
     }
   }
+
+  if (request.method === "POST") {
+    try {
+      const nurseData = request.body;
+      const newNurse = await Nurse.create(nurseData);
```
Comment (moonwave99) [inline pages/api/nurses/[id].js:41]: `const newNurse = await Nurse.create(request.body);` is enough ^^

File: pages/api/nurses/index.js [hunk 3]
```diff
@@ -18,14 +18,4 @@ export default async function handler(request, response) {
...
-  if (request.method === "POST") {
-    try {
-      const nurseData = request.body;
-      await Nurse.create(nurseData);
-      response.status(201).json({ message: "Nurse added!🍻" });
-    } catch (error) {
-      response.status(400).json({ message: error.message });
-    }
-  }
-  return response.status(405).json({ error: "Method not allowed" });
```
Comment (moonwave99) [inline pages/api/nurses/index.js:30]: The POST handler should stay in `pages/api/nurses/index.js`, is there a specific reason why you moved to `pages/api/nurses/[id].js`?

Comment (MiketheBite) [reply pages/api/nurses/index.js:30]: No reason, simply a trial/error attempt to solve the problem that I had

**Pull request merged**